package.json

```diff
@@ -29,16 +34,19 @@
   "devDependencies": {
     "babel-cli": "^6.7.7",
     "babel-core": "^6.7.7",
+    "babel-eslint": "^6.1.1",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-stage-0": "^6.5.0",
     "babel-register": "^6.7.2",
     "chai": "^3.0.0",
     "eslint": "^2.8.0",
     "eslint-config-airbnb-base": "^1.0.3",
     "eslint-plugin-import": "^1.5.0",
-    "lodash.isplainobject": "^3.2.0",
+    "eslint-watch": "^2.1.13",
     "flux-standard-action": "^0.6.0",
-    "mocha": "^2.2.5"
+    "lodash.isplainobject": "^3.2.0",
```

Comment:
I don't think this is actually being used - maybe we should get rid of it now as getting rid of unused `devDependencies` might fall under refactoring the local build/dev workflow?